scenarios/node_hog.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: litmuschaos.io/v1alpha1
+description:
+  message: |
+    Give a cpu spike on a node belonging to a deployment
+kind: ChaosExperiment
```

Comment:
This CR is being installed by https://github.com/openshift-scale/kraken/pull/33/files#diff-1f4248c56fcac8c6a639e9812e8fd51cb324a8cbc3aa79b34a7d38aa90451775R23 which installs all of the experiments.
This file can be removed and the env vars should be set via the ChaosEngine resource.

Reply:
Removed file